packages/components/src/visually-hidden/style.scss

```diff
@@ -0,0 +1,30 @@
+.components-visually-hidden {
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	-webkit-clip-path: inset(50%);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+}
+
+.components-visually-hidden:focus {
+	background-color: #eee;
+	clip: auto !important;
+	clip-path: none;
+	color: #444;
+	display: block;
+	font-size: 1em;
+	height: auto;
+	left: 5px;
+	line-height: normal;
+	padding: 15px 23px 14px;
+	text-decoration: none;
+	top: 5px;
+	width: auto;
+	z-index: 100000;
```

Comment:
In the context of Gutenberg, we probably should use a special helper which processes all `z-indexes`:

https://github.com/WordPress/gutenberg/blob/7515397fef469e8cbdaf68162d6d6e1f2a5f9aa1/packages/base-styles/_z-index.scss#L127-L133